client/blocks/dismissible-card/README.md

```diff
@@ -62,4 +62,15 @@ Note that we prefix this value with 'dismissible-card-' to avoid namespace colli
 	<tr><td>Default</td><td><code>undefined</code></td></tr>
 </table>
 
-When true, clicking on the cross will dismiss the card for the current page load.
\ No newline at end of file
+When true, clicking on the cross will dismiss the card for the current page load.
+
+
+### `dismissSeconds`
+
+<table>
+	<tr><td>Type</td><td>Number</td></tr>
+	<tr><td>Required</td><td>No</td></tr>
+	<tr><td>Default</td><td><code>undefined</code></td></tr>
+</table>
+
+When set, will dismiss the card for the amount of seconds
```

Comment:
Super minor, but this sentence is missing a trailing dot. We may also want to specify that this option will only be taken into account if  `temporary` is not set to `true`.

Reply:
Actually, what about reusing `temporary` instead of introducing a new prop? We are dealing with two interrelated concepts here. Thus `temporary` could accept either a boolean, or a number of seconds.